### PR TITLE
Update the content of Package.md and README.md file

### PR DESCRIPTION
**Pull request by ayeshLK**
Related to [#2657](https://github.com/ballerina-platform/ballerina-standard-library/issues/2657)

## Examples
N/A

## Checklist
- [X] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,11 +7,332 @@ Ballerina Email Library
   [![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/email.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Femail)
   [![codecov](https://codecov.io/gh/ballerina-platform/module-ballerina-email/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerina-email)
 
-The Email package is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
+This module provides APIs to perform email operations such as sending and reading emails using the SMTP, POP3, and IMAP4 protocols.
```
Comment (daneshk) [inline README.md:10]:
```suggestion
This library provides APIs to perform email operations such as sending and reading emails using the SMTP, POP3, and IMAP4 protocols.
```

File: ballerina/Package.md [hunk 2]
```diff
@@ -1,8 +1,331 @@
 ## Package Overview
 
-The `email` library is one of the standard library packages of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
+This module provides APIs to perform email operations such as sending and reading emails using the SMTP, POP3, and IMAP4 protocols.
```
Comment (BuddhiWathsala) [inline ballerina/Package.md:3]:
```suggestion
This package provides APIs to perform email operations such as sending and reading emails using the SMTP, POP3, and IMAP4 protocols.
```

File: ballerina/Package.md [hunk 3]
```diff
@@ -1,8 +1,331 @@
...
 
-It provides an implementation for sending and receiving emails with SMTP, POP3, and IMAP4 protocols.
+### Client
+
+This module supports the following three client types.
```
Comment (BuddhiWathsala) [inline ballerina/Package.md:7]:
```suggestion
This package supports the following three client types.
```

**Comment by daneshk** [comment]
Shall we change the icon as well

File: README.md [hunk 4]
```diff
@@ -7,11 +7,332 @@ Ballerina Email Library
...
 
-It provides implementation for sending emails over SMTP and receiving (with client and listener modes) over POP3 and IMAP4 protocols.
+### Client
 
-For more information, go to the [`email` package](https://docs.central.ballerina.io/ballerina/email/latest).
+This module supports the following three client types.
```
Comment (BuddhiWathsala) [inline README.md:14]:
```suggestion
This library supports the following three client types.
```

File: ballerina/Package.md [hunk 5]
```diff
@@ -1,8 +1,331 @@
...
+    remote function onError(email:Error emailError) {
+        io:println("Error while polling for the emails: " + emailError.message());
+    }
+
+}
+```
+
+### Security and Authentication
+
+The `email` module supports both the TLS/SSL and STARTTLS as transport-level security protocols.
```
Comment (BuddhiWathsala) [inline ballerina/Package.md:166]:
```suggestion
The `email` package supports both the TLS/SSL and STARTTLS as transport-level security protocols.
```

File: README.md [hunk 6]
```diff
@@ -7,11 +7,332 @@ Ballerina Email Library
...
+    remote function onError(email:Error emailError) {
+        io:println("Error while polling for the emails: " + emailError.message());
+    }
+
+}
+```
+
+### Security and Authentication
+
+The `email` module supports both the TLS/SSL and STARTTLS as transport-level security protocols.
```
Comment (BuddhiWathsala) [inline README.md:173]:
```suggestion
The `email` library supports both the TLS/SSL and STARTTLS as transport-level security protocols.
```

**Review by daneshk — APPROVED** [review]

**Pull request merged**